mutiny-core/src/nodemanager.rs

```diff
@@ -677,11 +677,10 @@ impl NodeManager {
 
     pub async fn label_peer(
         &self,
-        peer: &PublicKey,
+        node_id: &NodeId,
```

Comment:
Everything using gossip was a NodeId so I switched it everything there so we don't need to do unnecessary conversions. We are currently using PublicKey to identify our nodes which is fine I think because that's the type they return to us, if we used NodeId we'd just be adding an unnecessary conversion.